src/components/inputs/InputTextArea/textarea.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/theme";
+
+.text-area--container {
+  .label {
+    font-size: $font_size_1_4;
+    line-height: $line_height_2_2;
+    margin-bottom: $spacing_0_4;
+    font-weight: $font_semi_bold;
+    color: #3d527b;
```

Comment:
@vasilen-7DIGIT I have seen many colors which are not included in the '_variables.scss'. Should we in this scenarios add them to the global styling with appropriate naming. What is your opinion?